user-interface/frontend/themes/datamanager/components/dialog.css

```diff
@@ -202,12 +202,30 @@ vaadin-dialog-overlay::part(title) {
   align-items: baseline;
 }
 
+.edit-project-dialog .contact-field .prefill-input-fields {
+  line-height: 0;
+  display: flex;
+  gap: 1em;
+  align-items: baseline;
+}
+
 .contact-field .contact-selection,
 .contact-field .input-fields,
 .contact-field .input-fields > * {
   width: 100%;
 }
 
+.contact-field .prefill-input-fields {
+  line-height: 0;
+  display: flex;
+  gap: 1em;
+  align-items: baseline;
+}
+
+.contact-field .prefill-input-fields .contact-self-select {
+  width: 130px;
```

Comment:
please do not use absolute pixel values. Depending on the screen those appear at different sizes.